### ISSUE TITLE
Failure to open at login

### ISSUE THREAD

**Issue by nolorfin** [FruityRazer/FruityUI-Swift#51]
The app fails to load at boot after ticking the "open at login" option. The box is not ticked after rebooting.

<img width="1072" alt="Screenshot 2021-11-21 at 01 43 09" src="https://user-images.githubusercontent.com/72673748/142745036-4d7a6d89-0e60-40b9-b989-4b113a586939.png">

**Comment by edualm** [comment]
Interesting, I don't see why that would happen. 

Is the app on the Applications folder?

Also, would you mind sharing your macOS version?

**Comment by nolorfin** [comment]
Yeah, the app was on the Applications folder. My OS is the latest, or it should be: 12.0.1.

I fixed the issue by manually adding FruityUI to the list of Login items. I wonder if it should have been there in the first place (I recently moved back to MacOS after a 7-year stint on Windows).

Maybe the problem was that I changed the location of the home folder from the internal to an external SSD, and back. I don't recall at which point in the process I installed the app, but maybe something broke down on the way.

In any case, the problem is now gone for me thanks to the workaround above, though it may come up to other users in the future.
